kong/plugins/ldap-auth/access.lua

```diff
@@ -103,6 +104,25 @@ local function ldap_authenticate(given_username, given_password, conf)
   return is_authenticated, err
 end
 
+
+local function cache_key(conf, username, password)
+  local prefix = ldap_config_cache[conf]
+  if not prefix then
+    prefix = fmt("%s:%u:%s:%s:%u",
+                 lower(conf.ldap_host),
+                 conf.ldap_port,
+                 conf.base_dn,
+                 conf.attribute,
+                 conf.cache_ttl)
+
+    ldap_config_cache[conf] = prefix
+  end
+
+  local suffix = fmt("%s:%s:%s", prefix, username or "", password or "")
```

Comment:
Are there known cases where these variables are nil, and those we need the `or “”`? Or just being defensive?

Reply:
Perhaps not, just being defensive, I can remove them.